### ISSUE TITLE
Add creation methods to CustomGrid

### ISSUE THREAD

**Issue by Sharparam** [CoraleStudios/Colore#26]
We should add some helper creation methods to the `CustomGrid` struct, to easily create empty grids, or grids filled with a specific color, something like:

```c#
var myGrid = CustomGrid.Create(); // Creates an empty grid (filled with the color black)
var redGrid = CustomGrid.Create(Color.Red); // Creates a grid with every cell set to red
```

**Comment by Sharparam** [comment]
Empty grid might already work, as currently it will initialize the arrays, and since they are struct arrays, they should already initialize with black colors in all positions. **Needs confirmation**

**Comment by Sharparam** [comment]
This was implemented in PR #33.
